src/main/kotlin/no/nav/familie/baks/infotrygd/feed/config/KafkaErrorHandler.kt

```diff
@@ -0,0 +1,89 @@
+package no.nav.familie.baks.infotrygd.feed.config
+
+import org.apache.kafka.clients.consumer.Consumer
+import org.apache.kafka.clients.consumer.ConsumerRecord
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.core.task.SimpleAsyncTaskExecutor
+import org.springframework.kafka.listener.CommonContainerStoppingErrorHandler
+import org.springframework.kafka.listener.MessageListenerContainer
+import org.springframework.stereotype.Component
+import java.time.Duration
+import java.util.concurrent.Executor
+import java.util.concurrent.atomic.AtomicInteger
+import java.util.concurrent.atomic.AtomicLong
+
+@Component
+class KafkaErrorHandler : CommonContainerStoppingErrorHandler() {
```

Comment:
Personlig så synes jeg at denne error handleren fungerer bra i mottaks-applikasjon, men at 3 timers venting er litt i overkant for de fleste. I statistikk så har vi en enklere variant
**https://github.com/navikt/familie-ba-statistikk/blob/master/src/main/kotlin/no/nav/familie/ba/statistikk/config/RestartingKafkaErrorHandler.kt**

Reply:
fikset